SConstruct

```diff
@@ -816,6 +816,28 @@ libraries = {
 
 	"GafferTractorUITest" : {},
 
+	"GafferVDB" : {
+		"envAppends" : {
+			"LIBS" : [ "Gaffer", "GafferScene", "Half", "openvdb" ],
+		},
+		"pythonEnvAppends" : {
+			"LIBS" : [ "GafferScene", "GafferVDB", "openvdb" ],
+		}
+	},
+
+	"GafferVDBUI" : {
+		"envAppends" : {
+			"LIBS" : [ "Gaffer", "GafferScene", "GafferSceneUI", "IECoreGL$CORTEX_LIB_SUFFIX", "GafferVDB", "openvdb" ],
```

Comment:
I need to add "Half" here, to get things linking on the Mac :

https://github.com/johnhaddon/gaffer/commit/8964283be66512e62306cefd9b25dab9a952abaa